src/libServer/IsolatedServer.cpp

```diff
@@ -372,6 +372,12 @@ void IsolatedServer::BindAllEvmMethods() {
                            "param01", jsonrpc::JSON_STRING, "param02",
                            jsonrpc::JSON_STRING, NULL),
         &LookupServer::GetEthTransactionByBlockNumberAndIndexI);
+
+    AbstractServer<IsolatedServer>::bindAndAddMethod(
+        jsonrpc::Procedure("eth_recoverTransaction",
```

Comment:
Can you point me where you found that it should be called `eth_recoverTransaction`? In geth source code?